rclcpp_components/test/test_component_manager_api.cpp

```diff
@@ -143,6 +143,84 @@ TEST_F(TestComponentManager, components_api)
     EXPECT_EQ(result.get()->unique_id, 0u);
   }
 
+  {
+    // Remap rules
+    auto request = std::make_shared<composition_interfaces::srv::LoadNode::Request>();
+    request->package_name = "rclcpp_components";
+    request->plugin_name = "test_rclcpp_components::TestComponentFoo";
+    request->node_name = "test_component_remap";
+    request->remap_rules.push_back("alice:=bob");
+
+    auto result = client->async_send_request(request);
+    auto ret = exec->spin_until_future_complete(result, 5s);  // Wait for the result.
```

Comment:
can you do something like this ? instead of waiting a fix amount of time
```
  if (rclcpp::spin_until_future_complete(node, result) ==
    rclcpp::executor::FutureReturnCode::SUCCESS)
  {
     Expect_*()
  } else {
    FAIL();
  }
```

Reply:
This matches the style of the previously existing tests. The 5s is only a timeout, so this finishes very quickly when it's running correctly.